src/orquestra/sdk/_base/_driver/_client.py

```diff
@@ -259,13 +270,19 @@ def create_workflow_def(
     def list_workflow_defs(
         self, page_size: Optional[int] = None, page_token: Optional[str] = None
     ) -> Paginated[WorkflowDef]:
-        """
-        Lists all known workflow definitions
+        """Lists all known workflow definitions.
+
+        Args:
+            page_size: Maximum number of results returned in a single page.
+            page_token: Token for the requested page of the list results.
+                If omitted the first page will be returned.
 
         Raises:
-            orquestra.sdk._base._driver._exceptions.InvalidTokenError
-            orquestra.sdk._base._driver._exceptions.ForbiddenError
-            orquestra.sdk._base._driver._exceptions.UnknownHTTPError
+            orquestra.sdk._base._driver._exceptions.InvalidTokenError: when the
+                authorization token is rejected by the remote cluster.
+            orquestra.sdk._base._driver._exceptions.ForbiddenError: TODO
```

Comment:
I couldn't immediately see a good description of when this is raised, any suggestions?

Reply:
[2febed5](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/316/commits/2febed57570e8e817479bb98b34f3c37cd35ba55)